### PR TITLE
feat(pt): support `training/profiling` argument in PT

### PR DESCRIPTION
**Pull request by njzjz**
## Summary by CodeRabbit

- **New Features**
	- Added profiling functionality with parameters for enabling profiling and exporting data to a Chrome trace file.
  
- **Documentation**
	- Updated documentation for profiling-related arguments to clarify export options for performance analysis.

### CODE REVIEW

**Review by wanghan-iapcm — COMMENTED** [review]
im confused by the relationship between the options `enable_profiler` and `profiling`. could you please explain more in the doc?

File: deepmd/pt/train/training.py [hunk 1]
```diff
@@ -699,6 +699,8 @@ def warm_up_linear(step, warmup_steps):
         self.tensorboard_log_dir = training_params.get("tensorboard_log_dir", "log")
         self.tensorboard_freq = training_params.get("tensorboard_freq", 1)
         self.enable_profiler = training_params.get("enable_profiler", False)
+        self.profiling = training_params["profiling"]
+        self.profiling_file = training_params["profiling_file"]
```
[the remark this answers is not in the document]
Comment (njzjz) [reply deepmd/pt/train/training.py]: Fixed in https://github.com/deepmodeling/deepmd-kit/pull/3897/commits/7643cfbc445bf72856f97c3ec693b31223302bee.

**Comment by njzjz** [comment]
> im confused by the relationship between the options `enable_profiler` and `profiling`. could you please explain more in the doc?

I updated the documentation. While the behavior of the two options never changed, I am wondering whether we should give a more clear name.

**Review by iProzd — APPROVED** [review]

**Review by wanghan-iapcm — APPROVED** [review]
Shall we deprecate the `profiling` option in the future?

**Pull request merged**